docs/about/index.rst

```diff
@@ -1,13 +1,37 @@
 .. _about:
 
-About
-=====
+About Fairlearn
+===============
 
 .. _mission:
 
 Mission
 -------
 
+Fairlearn is an open source project that seeks to empower data scientists and
+developers to assess and improve fairness of AI systems.
+The project aspires to include:
+
+- A Python library of fairness metrics and mitigation algorithms
+- Tools for fairness assessment (e.g., plotting, dashboards, etc.)
+- Educational materials (comprehensive user guide, detailed case studies,
+  Jupyter notebooks, white papers, etc.)
+
+Development of Fairlearn is firmly grounded in the understanding that fairness
+in AI systems is a sociotechnical challenge.
+Because there are many complex sources of unfairness --- some societal and
+some technical --- it is not possible to fully "debias" a system or to
+guarantee fairness.
+Our goal is to enable humans to assess fairness-related harms, review the
```

Comment:
Is there a specific reason for the word "humans" here, instead of e.g.., people/practitioners/etc.?